android/java/res/layout/new_tab_page_layout.xml

```diff
@@ -25,6 +25,12 @@
         android:layout_height="match_parent"
         android:contentDescription="@null"/>
 
+    <FrameLayout
+        android:id="@+id/bg_background_view"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+    </FrameLayout>
```

Comment:
shouldn't it to be `android:visibility="gone"`?

Reply:
Fixed